### PR TITLE
T-Watch 2020 V2 & V3 support

### PR DESCRIPTION
**Pull request by JohnySeven**
This PR allows user to compile firmware for V2 and V3 variants of T-Watch 2020 hardware. There is some issue with V2 as it has enhanced vibration motor control - for now it doesn't work. Will need to get it working when my V2 device arrives from China.

@ba58smith if you have some time could you verify compilation for all variants? Also I've changed how code is being compiled via PlatformIO. From now on it's necessary to switch platformio project environment:
image

I've already published V2 and V3 as BETA into new designer.

### CODE REVIEW

**Comment by ba58smith** [comment]
@JohnySeven, I will try compiling all 3 versions when our guests leave, and I have a few hours.

**Comment by ba58smith** [comment]
All 3 "env" variations built without a problem. Good job!

File: src/hardware/hardware.cpp [hunk 1]
```diff
@@ -18,8 +18,12 @@ const char *HW_TAG = "HW";
 Hardware::Hardware() : Configurable("/config/hardware")
 {
     load();
+#ifdef LILYGO_WATCH_2020_V2
+
```
Comment (ba58smith) [inline src/hardware/hardware.cpp:22]: Did you mean to do something with this #ifdef?

**Comment by JohnySeven** [comment]
With last commits support for V2 is much better. Merging into main.

**Pull request merged**